lms/static/js/student_account/views/FormView.js

```diff
@@ -185,32 +176,31 @@
 
                 saveError: function(error) {
                     this.errors = ['<li>' + error.responseText + '</li>'];
-                    this.setErrors();
+                    this.renderErrors(this.defaultFormErrorsTitle, this.errors);
                     this.toggleDisableButton(false);
                 },
 
-                setErrors: function() {
-                    var $msg = this.$errors.find('.message-copy'),
-                        html = [],
-                        errors = this.errors,
-                        i,
-                        len = errors.length;
-
-                    for (i = 0; i < len; i++) {
-                        html.push(errors[i]);
-                    }
+                renderErrors: function(title, errorMessages) {
+                    this.renderFormFeedback(this.formErrorsTpl, {
+                        context: {
+                            title: title,
+                            messagesHtml: HtmlUtils.HTML(errorMessages.join(''))
+                        }
+                    });
+                },
 
-                    $msg.html(html.join(''));
+                renderFormFeedback: function(template, context) {
+                    this.clearFormFeedback();
 
-                    this.element.show(this.$errors);
+                    HtmlUtils.append(this.$formFeedback, HtmlUtils.template(template)(context));
```

Comment:
This code would be easier to read if you broke the template rendering into 2 stages.